gazebo/gazebo_shared.cc

```diff
@@ -27,6 +29,11 @@
 void gazebo_shared::printVersion()
 {
   fprintf(stderr, "%s", GAZEBO_VERSION_HEADER);
+
+  char *ignoreEol = getenv("GAZEBO_SUPPRESS_EOL_WARNING");
```

Comment:
consider using the cross-platform [common::getEnv](https://github.com/gazebosim/gazebo-classic/blob/9aefb5843b4358c0189009538d601eae55bb5d25/gazebo/common/CommonIface.hh#L86) here and in MainWindow

Reply:
According to https://github.com/search?q=repo%3Agazebosim%2Fgazebo-classic+%2F%28%3F-i%29getEnv%2F&type=code , there are only 8 usages of this function in Gazebo Classic. Is it worth it?

Reply:
Not critical for 99% of the users probably but given that we are already using it in other places and it is the right way of doing it my vote is to use it yes. Thanks Martin.

Reply:
Okay, changed.